project-bundle-migration/plugin.json

```diff
@@ -0,0 +1,13 @@
+{
+    "id" : "Project_Bundle_Migration",
```

Comment:
I would use a more "standard" ID sticking with the plugin name, so "dataiku-project-bundle-migration" for instance